### DISCUSSION TITLE
Noob questions around looping and hotkeys

### DISCUSSION THREAD

**Discussion by holzon** [helgoboss/helgobox#1345] [General]
Hi!
Just got started with Playtime, really happy to have some options around pick-up beats in a clip launcher finally!!
I couldn't find a hotkey for quickly duplicating clips within its column, I was expecting to be able to shift+drag clips to quickly make copies, but couldn't find anything. Is there such a modifier already?

Was also playing around with pick-up beat functionality yesterday but couldn't really wrap my head around making my clips loop nicely with it. To sum how I have been thinking to work is:
1. Record a metal riff including pre-beat audio and tail
2. Set it to start a wee bit early so as not to cut off first hit of audio
3. Set it to loop and cross fade, but I would like it to hit the beat, it does not need to go back to pre-beat position when looping. I did not manage to set this up
(4. If one could dream, it would play the tail when writing to arrangement and stopping the clip, but this might be fantasy 😁 )
Do you need to chain up rows to achieve such a loop setup or is there a way with current workflow that I'm not seeing?

Also, I'm constantly wanting to drag within the clip audio preview window to adjust start and downbeat markers, that would be so nice, and maybe grids or ticks to show beats in that window 🙏

**Comment by helgoboss** [comment]
Hi!

I think you'd want to vote for feature request #1049. The current "Downbeat" functionality on its own also repeats the downbeat itself.

Considering the waveform window stuff, #1012 is the feature request for you.

**Reply by holzon** [reply]
Perfect, thanks for taking the time to help out! 🙏
